backend/app/models/openai_conversation.py

```
@@ -0,0 +1,58 @@
+
+class OpenAIConversation(OpenAIConversationBase, table=True):
+    __tablename__ = "openai_conversation"
+
+    id: int = Field(default=None, primary_key=True)
+    inserted_at: datetime = Field(default_factory=now, nullable=False)
+    updated_at: datetime = Field(default_factory=now, nullable=False)
+    is_deleted: bool = Field(default=False, nullable=False)
+    deleted_at: Optional[datetime] = Field(default=None, nullable=True)
+
+    # Relationships
+    project: "Project" = Relationship(back_populates="openai_conversations")
```

Ruff (0.12.2)

Undefined name `Project`

(F821)